tox.ini

```diff
@@ -113,7 +114,30 @@ commands =
 extras =
     test
 commands =
-    pytest {posargs} --disable-warnings --log-cli-level=info --log-cli-format="%(asctime)s [%(levelname)8s] %(name)s:%(lineno)s %(message)s" --log-date-format="%Y-%m-%d %H:%M:%S" --cov-report=term --cov-report=xml --cov-config=.coveragerc --cov={envsitepackagesdir}/pudl test/etl_test.py test/glue_test.py test/fast_output_test.py test/interim_etl_test.py test/zenodo_integration.py
+    pytest \
+      --cov-report=term --cov-report=xml --cov-config=.coveragerc \
+      --cov={envsitepackagesdir}/pudl \
+      src/pudl
+    pytest {posargs} \
+      --disable-warnings --log-cli-level=info \
+      --log-cli-format="%(asctime)s [%(levelname)8s] %(name)s:%(lineno)s %(message)s" \
+      --log-date-format="%Y-%m-%d %H:%M:%S" \
+      --cov-report=term --cov-report=xml --cov-config=.coveragerc \
+      --cov={envsitepackagesdir}/pudl \
+      --cov-append \
+      test/etl_test.py \
+      test/glue_test.py \
+      test/fast_output_test.py \
+      test/interim_etl_test.py \
+      test/zenodo_integration.py
```

Comment:
omg, why did I not realize we could write it like this before. This is so much nicer.